modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/background/task/CTPublishBackgroundTaskExecutor.java

```diff
@@ -306,6 +338,22 @@ public void setAopProxy(Object aopProxy) {
 		_backgroundTaskExecutor = (BackgroundTaskExecutor)aopProxy;
 	}
 
+	private long _getAdminUserId(long companyId) throws PortalException {
+		Role role = _roleLocalService.getRole(
+			companyId, RoleConstants.ADMINISTRATOR);
+
+		long[] userIds = _userLocalService.getRoleUserIds(role.getRoleId());
+
+		if (userIds.length == 0) {
+			throw new NoSuchUserException(
+				StringBundler.concat(
+					"No user exists in company ", companyId, " with role ",
+					role.getName()));
+		}
```

Comment:
This check is not needed because there will always be at least one administrator in the company.